src/ui/component/notification/notification.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC } from 'react'
+import { useMemo, useCallback } from 'react'
+import { useAppStore } from '@/ui/store/appStore'
+import type { IconName } from '@/ui/component/icon/icon'
+import { Toast } from '@/ui/component/toast/toast'
+import type { NotificationType } from '@/domain/notification/entity'
+import { toEffectfulObject } from '@/util/effect'
+import { useNotificationStore } from '@/ui/store'
+import type { DismissNotificationInput } from '@/domain/notification/aggregate'
+
+export const Notification: FC = () => {
+  const { notifications, dismissNotification } = toEffectfulObject(
+    useNotificationStore(state => ({
+      notifications: state.notifications,
+      dismissNotification: state.dismissNotification
+    }))
+  )
+  const theme = useAppStore(store => store.theme)
+  const notificationData = useMemo(() => notifications(), [notifications])
```

Comment:
This one is for me :laughing: 
Well, this tweet explains why: https://twitter.com/nikolaposa/status/1077327810627358721?s=20

Morover, imho, omitting the `get` prefix for getters in entities and APIs (a zustand store is exposed as such) allows the focus to be on the object's properties (which is the domain term) rather than its methods.

Reply:
Thanks for these explanation 👍
I think it's confusing to read `notifications()` because it's a mix of const and function (for me), but why not, I'll get used to it 😉